lesson 12/styles/style.css

```diff
@@ -1110,6 +1815,353 @@
         font-size: 15px;
     }
 
+    .homes__container {
+        padding: 40px 15px 56px 17px;
+    }
+
+    .homes__title {
+        font-size: 28px;
+        margin-bottom: 32px;
+    }
+
+    .homes__block {
+        column-gap: 16px;
+    }
+
+    .homes_img {
+        width: 100%;
+        max-width: 156px;
+        aspect-ratio: 1/1;
+        margin-bottom: 12px;
+    }
+
+    .homes_signature {
+        font-size: 12px;
+    }
+
+    .homes_text {
+        font-size: 12px;
+    }
+
+    .homes__pointer {
+        left: 96%;
+        top: 25%;
+        box-shadow: 2px 2px 2px #959597;
+        width: 28px;
+        height: 28px;
+        border-radius: 50%;
+        border: none;
+        padding: 4px 5px 8px 10px;
+    }
+
+    .polygon {
+        fill: #383838;
+        width: 5px;
+        height: 13px;
+    }
+
+    .homes__subblock-3 {
```

Comment:
Whu you didn't create on class subblock with property display:none and insteas of this you create nu,erous of classes, looks strange